### ISSUE TITLE
Don't require write acces on a repo to assign issues or label them

### ISSUE THREAD

**Issue by Luis-Fernando-Molina** [dear-github/dear-github#127]
We have several teams working on the same repo and right now all team leaders need to have write access to it in order to assign issues to their team members or label these issues. 

Each team work on their own fork and changes are pushed later upstream. 

It would be great is we could only give write access to a few core developer to the repo and not to all team leaders working on forks. We don't really see the need to attach an administrative task like assigning issues or labelling them to a permission of modifying the source code on the repository.

I have already reported this to github for months and I am still waiting for news if they are going to take it or not.

**Comment by aensley** [comment]
>We don't really see the need to attach an administrative task like assigning issues or labelling them to a permission of modifying the source code on the repository.

Yes, exactly! We have customer service reps who need access to the ticketing system this way. I don't like that they also have the ability to delete all of our code.
